charts/openshift-metering/templates/hive/hive-scripts-configmap.yaml

```diff
@@ -81,6 +81,22 @@ data:
     ln -s -f /hive-config/hive-log4j2.properties $HIVE_HOME/conf/hive-log4j2.properties
     ln -s -f /hive-config/hive-exec-log4j2.properties $HIVE_HOME/conf/hive-exec-log4j2.properties
 
+    # for databse configuration to mysql
+    # if a databse secret exists, check hive-site.xml, to ensure no values added yet. Inject with username and password stored
+    # in volume as database secret
+    [[ if -f /var/run/secrets/metastore-db-secrets ]];
+    then
+    faq'.configuration.property' /hive-config/hive-site.xml
+    faq'.configuration.property+=[
+    { name: "javax.jdo.option.ConnectionUserName", value: "/var/run/secrets/metastore-db-secrets/username"}
+    ]'
+    faq'.configuration.property+=[
+    { name: "javax.jdo.option.ConnectionPassword", value: "/var/run/secrets/metastore-db-secrets/password"}
+    ]'
+    /hive-config/hive-site.xml > /hive-config/hive-site-tmp.xml
+    mv -f /hive-config/hive-site-tmp.xml /hive-config/hive-site.xml
+    fi
```

Comment:
A couple of things:
1. the `if` statement needs to be outside of the square brackets.
2. the `then` statement should be after the closing square brackets, e.g. `]]; then`.
3. this is mis-idented.